CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Added
 
+- (crate internal) Added `-D warnings` flags to workflow to help catch some documentation errors 
```

Comment:
```suggestion
- CI: `-D warnings` flags to cargo workflows.
```

Reply:
I'd personally leave "to catch documentation errors".